zerver/views/typing.py

```diff
@@ -21,10 +22,27 @@ def send_notification_backend(
         "type", str_validator=check_string_in(VALID_MESSAGE_TYPES), default="private"
     ),
     operator: str = REQ("op", str_validator=check_string_in(VALID_OPERATOR_TYPES)),
-    user_ids: List[int] = REQ("to", validator=check_list(check_int)),
+    notification_to: List[int] = REQ("to", validator=check_list(check_int)),
+    topic: Optional[str] = REQ("topic", default=None),
 ) -> HttpResponse:
-    if len(user_ids) == 0:
-        return json_error(_("Missing parameter: 'to' (recipient)"))
+    to_length = len(notification_to)
+
+    if to_length == 0:
+        return json_error(_("Empty 'to' list"))
+
+    if message_type == "stream":
+        if to_length > 1:
+            return json_error(_("Multiple stream IDs not supported"))
+
+        if topic is None:
+            return json_error(_("Missing parameter 'topic'"))
```

Comment:
To what extent do these error strings match the errors the `send_message` API gives for the same parameter error?  Ideally, we'd try to match that to avoid making translators do extra work for this endpoint.

Reply:
For the empty `to` list I think it'd be better to go with `Empty 'to' list` since the `send_message` says `Message must have recipients` for private and `Missing stream` for stream types.

Changed the other two to `Cannot send to multiple streams` and `Missing topic` which match with those of `send_message`.